### PR TITLE
AEROGEAR - 3508 - add docker version to oc cluster up doc links

### PR DESCRIPTION
**Pull request by ciaranRoche**
## JIRA 
https://issues.jboss.org/browse/AEROGEAR-3508
## Screenshot
update-prereq
## Additional Info
On the assumption of changes from https://github.com/aerogear/mobile-docs/pull/130 getting merged, I changed around the format to add the docker version to the oc cluster up note. Just looking for feedback on this.

### CODE REVIEW

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 1]
```diff
@@ -12,13 +12,12 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
...
 
-* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] (tested with version 2.4, but should work with version 2.3 or later)
-
 * link:https://www.openshift.org/download.html[OpenShift client tools] version 3.9 or later
 +
 WARNING: OpenShift oc executable must be located in a system location that is known to all shells (e.g. /usr/local/bin)
 
-* Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and depending on your OS this is downloaded from different places:
++
+NOTE: Ensure link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] is working. Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and depending on your OS this is downloaded from different places:
```
Comment (johnfriz) [inline modules/ROOT/pages/installing-mobile-services.adoc]: Suggested change:

Split into 2 separate notes:

NOTE: Ensure that you can run oc cluster up with no errors before moving on to the installation of AeroGear services.

NOTE: Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and AeroGear Mobile Services. This version may be different from the version listed on the oc cluster up documentation. You must ensure you are running with this version of docker and that oc cluster works without errors before moving on. Depending on your OS this is downloaded from different places:

**Comment by ciaranRoche** [comment]
Updated layout @johnfriz mind having another look. Screenshot updated in the comment also to reflect the change.

**Comment by johnfriz** [comment]
@ciaranRoche - The wording of "Ensure that you can run oc cluster up with no errors before moving on to the installation of AeroGear services." is not quite right.

It currently says "Ensure that you can run oc cluster up *services. [oc cluster up]* with no errors before moving on to the installation of AeroGear services."

**Comment by ciaranRoche** [comment]
@johnfriz wording and screenshot updated

**Pull request merged**